src/register/data/tests/reducers.test.js

```diff
@@ -63,6 +64,37 @@ describe('Registration Reducer Tests', () => {
       },
     );
   });
+  it('on regirstration success call', () => {
+    const payload = {
+      redirectUrl: 'http://localhost:18000/dashboard',
+      success: true,
+    };
+    const action = {
+      type: REGISTER_NEW_USER.SUCCESS,
+      payload,
+    };
+
+    expect(reducer(defaultState, action)).toEqual(
+      {
+        ...defaultState,
+        registrationResult: payload,
+      },
+    );
+  });
+
+  it('should set the registration call and set the registration error object empty', () => {
```

Comment:
Could you elaborate the purpose of this test?

Reply:
When we initiate the call after submitting the button then this flow triggers

Reply:
addressed